src/suites/cts/buffers/map_oom.spec.ts

```diff
@@ -0,0 +1,31 @@
+export const description = ``;
+
+import { TestGroup } from '../../../framework/index.js';
+import { GPUTest } from '../gpu_test.js';
+
+const bufferDesc: GPUBufferDescriptor = {
+  size: Number.MAX_SAFE_INTEGER,
+  usage: 2, // GPUBufferUsage.MAP_WRITE (TODO: not in scope during test crawling in node)
```

Comment:
Would it work if we just made this a function `const getBufferDesc = () => ({ size, usage: MAP_WRITE })` ?

Reply:
Good idea.